src/t_hash.c

```diff
@@ -1839,7 +1910,7 @@ static ExpireAction hashTypeActiveExpire(eItem _hashObj, void *ctx) {
                 .now = commandTimeSnapshot(),
                 .itemsExpired = 0};
 
-        listpackExExpire(hashObj, &info);
+        listpackExExpire(hashObj, &info, &server.stat_expired_hash_fields);
```

Comment:
this allows listpackExExpire() to be more independent.
```suggestion
        listpackExExpire(hashObj, &info);
        server.stat_expired_hash_fields += info.itemsExpired;
```